src/pages/City/AddAdministratorModal/AddAdministratorModal.tsx

```diff
@@ -243,11 +231,13 @@ const AddAdministratorModal = (props: Props) => {
         showDisable(admin);
       } else if (
         admin.adminType.adminTypeName === "Голова СПР" ||
-        admin.adminType.adminTypeName === "Член СПР"
+        admin.adminType.adminTypeName === "Член СПР" ||
```

Comment:
Maybe it is best to add this strings to Roles enum? You use it just below, why not here? Its is also best to check if this roles even exist and being used somewhere.

Reply:
> Maybe it is best to add this strings to Roles enum? You use it just below, why not here? Its is also best to check if this roles even exist and being used somewhere.

1) Moving roles to Enum should be addressed separately in another PR.
2) Not a priority